app-text/mupdf/mupdf-1.14.0-r3.ebuild

```diff
@@ -78,15 +79,11 @@ src_prepare() {
 
 _emake() {
 	# When HAVE_OBJCOPY is yes, we end up with a lot of QA warnings.
-	local have_libcrypto=no
```

Comment:
@hsoft why does it have this option then?

Reply:
Hum, yeah. it seems that I don't remember correctly... Should be fine then.